src/components/ScrollShadow.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react";
+
+export default function ScrollShadow({
+  children,
+}: {
+  children: React.ReactNode;
```

Comment:
`ReactNode` rather than `React.ReactNode`. Can't treeshake dependency easily like this. Just need to fix import if you make this change